src/app/pokemon.ts

```diff
@@ -0,0 +1,11 @@
+export class Pokemon{
+    id: number;
+    name: string;
+    form: string;
+    summary: string;
+    height: number;
+    category: string;
+    weight: number;
+    abilities: string;
+    gender: string;
```

Comment:
Why not make this optional?

Reply:
A class seemed like a better option than an interface